docs/Xamarin.Forms.Core/Xamarin.Forms/Slider.xml

```diff
@@ -182,8 +182,7 @@ namespace FormsGallery
       <Docs>
         <summary>Gets or sets the maximum selectable value for the Slider. This is a bindable property.</summary>
         <value>A double.</value>
-        <remarks>
-        </remarks>
+        <remarks>Must be data-bound before Value to avoid Value being set to default Maximum if Value is greater than default Maximum.</remarks>
```

Comment:
we still consider this behavior as room for improvements, documenting it is a good start, but doesn't help much.